streets-web/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "streets-web"
+version = "0.1.0"
+description = "Streets Map and Test Browser"
+readme = true
+repository = "https://github.com/a-b-street/osm2streets"
+license = "Apache 2.0"
+edition = "2021"
+
+[dependencies]
+console_log = { version = "0.2", optional = true, features = ["color"] }
+getrandom = { version = "0.2", features = ["js"] }
+gloo-utils = "0.1"
+leaflet = { git = "https://github.com/droogmic/leaflet-rs", branch = "polyline_get_bounds" }
+log = "0.4"
+#osm2lanes = { path = "../osm2lanes", features = ["overpass"] }
```

Comment:
I'll start working at dropping the dependencies you need out of the bottom of osm2lanes :)